### PR TITLE
Multiple users stacked dark mode

### PR DESCRIPTION
**Pull request by kellyeryan**
#### Runway Ticket URL

NUX-1328

I fixed a small bug with the badge kit to make this kit work properly.

#### Screens

BUG:

Screen Shot 2020-08-13 at 9 37 28 AM

Final Screenshots:

Screen Shot 2020-08-13 at 9 54 20 AM

Screen Shot 2020-08-13 at 9 54 05 AM

#### Breaking Changes

Only additive changes.

#### Checklist:

- [X] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [X] **SCREENSHOT** Please add a screen shot or two
- [X] **SPECS** Please cover your changes with specs
- [X] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: app/pb_kits/playbook/pb_badge/_badge.jsx [hunk 1]
```diff
@@ -32,12 +32,8 @@ const Badge = (props: BadgeProps) => {
...
   const dataProps = buildDataProps(data)
   const css = classnames(
-    className,
-    buildCss('pb_badge_kit', variant, {
-      rounded: rounded,
-    }),
-    globalProps(props)
-  )
+    buildCss('pb_badge_kit', variant, { rounded: rounded }), className,
+    globalProps(props))
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_badge/_badge.jsx:36]:
```suggestion
    buildCss('pb_badge_kit', variant, { rounded: rounded }), globalProps(props), className)
```

Reference https://github.com/powerhome/playbook/pull/929/files

File: app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.html.erb [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
   data: object.data,
-  class: object.classname) do %>
-  <%= pb_rails("avatar", props: object.users[0].merge({size: "xs", classname: "pb_multiple_users_stacked_item"}) ) %>
+  id: object.id) do %>
+  <%= pb_rails("avatar", props: object.users[0].merge({size: "xs", classname: "pb_multiple_users_stacked_item", dark: object.dark}) ) %>
 
   <% unless object.only_one %>
     <% if object.more_than_two %>
       <%= pb_rails("badge", props: {
+        dark: true,
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.html.erb:11]: Should this always be set to true?

File: app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.jsx [hunk 3]
```diff
@@ -9,33 +9,42 @@ import { Avatar, Badge } from '../'
...
   }
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
-  const css = buildCss({
-    'pb_multiple_users_stacked_kit': true,
-    single: onlyOne,
-  })
+  const classes = classnames(buildCss(
+    'pb_multiple_users_stacked_kit',
+    { single: onlyOne }), className, globalProps(props))
```
Comment (jasoncypret) [inline app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.jsx:39]:
```suggestion
  const classes = classnames(buildCss(
    'pb_multiple_users_stacked_kit',
    { single: onlyOne }), globalProps(props), className)
```

File: app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.html.erb (hunk 2 above)
Comment (kellyeryan) [reply app/pb_kits/playbook/pb_multiple_users_stacked/_multiple_users_stacked.html.erb:11]: No, you're right! I'll fix it.

**Pull request merged**